SPECS/python-click/python-click.spec

```diff
@@ -10,7 +10,7 @@
 Summary:        Simple wrapper around optparse for powerful command line utilities
 Name:           python-%{pypi_name}
 Version:        7.1.2
-Release:        5%{?dist}
+Release:        6%{?dist}
```

Comment:
There's quite a few things we need to do in the specs in terms of clean-up when we're adding them to the core specs. Unfortunately, I think we don't have it fully documented or automated yet - sorry about that. Anyway, what we'd need to do:

- Add the `Vendor` and `Distribution` tags.
- Lint the spec.
- Remove any non-Mariner-distro-specific macro. For instance we will never set and use macros like `%{fedora}` or `%{rhel}`. In case they are present, we tend to keep the bits of the spec, which would be left, if we assumed `%define fedora 33` (that's because in the initial package import we tend to stick to Fedora's configuration and adjust it later to Mariner's needs), and remove the bits of the spec for other distro-specific macros.
  Having said all that, I don't see any such macros in this spec, but that's just a general comment for all core additions.
- Use a single `Source0` line (drop the `#Source0` comment). In this case I think we can just drop the commented-out `Source0` tag.